### PR TITLE
Update outlook-for-ios-and-android-faq.md

### PR DESCRIPTION
**Pull request by peters-msft**
In Resource Forest environments when using Outlook for Mobile, the Cloud Cache Mailbox Sync synchronizes via the ActiveSync Interface (Cloud Cache Sync) and during Authentication the Account Forest also needs to Trust the Resource Forest (in opposite direction)

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/outlook-for-ios-and-android-faq.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ Yes, Outlook for iOS and Android supports certificate-based authentication for m
 
 - [Certificate-based authentication on Android](https://docs.microsoft.com/azure/active-directory/authentication/active-directory-certificate-based-authentication-android)
 
+### Q: What do I need to consider when my OnPremise Environment is running in a Resoruce Forest Scenario?
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/outlook-for-ios-and-android-faq.md:71]:
```suggestion
### Q: What do I need to consider when my on-premises environment is running in a resource forest scenario?
```

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/outlook-for-ios-and-android-faq.md [hunk 2]
```diff
@@ -68,6 +68,10 @@ Yes, Outlook for iOS and Android supports certificate-based authentication for m
...
+
+You have to make sure the Trust between the Resource Forest and the Account Forest is configured as a two-way Trust.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/outlook-for-ios-and-android-faq.md:73]:
```suggestion
You have to make sure the trust between the resource forest and the account forest is configured as a two-way trust.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**